.conda/environment.yml

```diff
@@ -17,7 +17,7 @@ dependencies:
   - tblib
   - thriftpy2
   - netifaces
-  - clang
-  - clangxx
+  - clang=11.1
+  - clangxx=11.1
```

Comment:
This is a suboptimal solution: sooner or later we have to change this again.

Reply:
I've created https://github.com/xnd-project/rbc/issues/334 to track this. Also, LLVM 11 works by lucky. Officially, llvmlite supports LLVM 10 on most architectures and LLVM 9 on arm64.

Reply:
Ok. Since both numba and clang are used in client-side, we should ensure that in conda environment that both use the same llvm version. I don't know if this can be achieved at environment.yml level. At least, we can acquire the clang version from rbc and then compare that with the llvm version that llvmlite uses.

Reply:
The environment.yml will always use the latest version of Numba. We can stick a LLVM version for clang(xx) that is compatible with Numba. What do you think?

Reply:
Sounds good. It likely means that we'll need to stick to some older version of clang that uses llvm that a numba version that rbc minimally supports.